### PR TITLE
Fix submodule URL

### PR DESCRIPTION
**Pull request by sbliven**
Update submodule URL from git.wincent.com to github & add some build documentation

### CODE REVIEW

**Comment by wincent** [comment]
Thanks.

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [submodule "buildtools"]
 	path = buildtools
-	url = git://git.wincent.com/buildtools.git
+	url = https://github.com/wincent/buildtools.git
 [submodule "WOPublic"]
 	path = WOPublic
-	url = git://git.wincent.com/WOPublic.git
```
Comment (wincent) [inline .gitmodules:6]: Grrrr. Annoying that Git isn't reliable on this host. I have monitoring set-up, believe it or not, but it somehow fails to notice that clone attempts time out. I need better monitoring...

**Comment by sbliven** [comment]
It seems like it might make sense to switch the default paths to github then, unless the wincent repos sometimes get ahead of the github ones.

**Comment by wincent** [comment]
Yeah, I am fine with moving the paths to GitHub. The git.wincent.com repo is the "official" one, but it gets mirrored out automatically to GitHub, GitLab and Bitbucket every hour (and sometimes I just `git push && git push github` when I don't want to wait for the mirror for whatever reason).

**Comment by wincent** [comment]
Whoops, accidentally clicked the "Close" button.

**Pull request closed without merging**